### DISCUSSION TITLE
full width Aggrid Table and custom button on last column

### DISCUSSION THREAD

**Discussion by awsomedesigner** [lowdefy/lowdefy#1198] [Questions and Help] [answered]
Hello Team

I am playing with aggrid. I am trying to do full-width table which is fits to parent container width. For this, I tried sizeColumnsToFit
but it's not working and applying with the table grid.

Also, I want to add two buttons on the last column to edit or delete that rows. Users can click on the edit button and can change anything from the data. If the user clicks on the delete button then data will remove from the database. I didn't find anything related to these in lowdefy documents

**Comment by Gervwyk** [comment, marked as answer]
For column full-width see: 
https://discord.com/channels/729696747261263962/837652812942540810/976483531247206430

**Reply by qin-zhuopu** [reply]
why you guys share answers on discord, are they secret?

**Reply by Gervwyk** [reply]
No, it is just that they as duplicated there. Here is a copy:

> Sam: I find applying flex: 1 to one or more of the columnDefs works well for me, as this makes the column expand to take up more space. You can apply it to only some columns if you only want them to expand, or you can add it in the defaultColDef so it applies to all columns. You can also do different numbers, then the columns will expand in those ratios.

You can join the Lowdefy community discord here: https://discord.gg/WmcJgXt

**Reply by qin-zhuopu** [reply]
thank you ! it works like charm
